06-files-input-and-output/csv_split.py

```diff
@@ -0,0 +1,133 @@
+# Copyright (c) 2016 Ami . All rights reserved
+import argparse
+import csv
+import os
+import sys
+
+def get_arguments():
+    current_directory = os.getcwd()
+    parser = argparse.ArgumentParser(description="Split CSV by every x rows")
+
+    parser.add_argument(
+        "-i",
+        action="store",
+        dest="input_file",
+        help="CSV input file",
+        required=True
+    )
+    parser.add_argument(
+        "-o",
+        action="store",
+        dest="output_path",
+        help="Output path - where to store the split CSV files",
+        default=current_directory
+    )
+    parser.add_argument(
+        "-r",
+        action="store",
+        dest="row_limit",
+        help="split by every x lines",
+        type=int,
+        required=True
+    )
+
+    if len(sys.argv) == 1:
+        parser.print_help()
+        return None
+    else:
+        return parser.parse_args()
+
+def check_if_file_exists(file_path):
+    if os.path.exists(file_path):
+        return True
+    else:
+        print("File not found:", file_path)
+        return False
+
+def validate_minimum_rows(csv_file_name, csv_file_num_lines, row_limit):
+    print(
+        "Validating file {file_name} has more than {row_limit} rows to split by...".format(
+            file_name=csv_file_name,
+            row_limit=row_limit,
+        ),
+        end=""
+    )
+
+    if csv_file_num_lines < row_limit:
+        print(
+            "Error: the file {file_name} contains {file_num_lines} lines, "
+            "it is smaller than the {row_limit} rows to split by".format(
+                file=csv_file_name,
+                file_num_lines=csv_file_num_lines,
+                row_limit= row_limit,
+            )
+        )
+        return False
+    else:
+        print("OK")
+        print("There are " + str(csv_file_num_lines) + " lines in the file")
+        return True
+
+def split_csv_to_chunks_by_rows(csv_data, row_limit):
+    print("Splitting CSV into chunks every " + str(row_limit) + " lines")
+
+    return split_list_in_groups_of(row_limit, csv_data)
+
+def split_list_in_groups_of(group_size, list_to_slice):
+    slices = list()
+
+    for index, item in enumerate(list_to_slice):
+        if index % group_size == 0:
+            current_slice = list()
+            current_slice.append(item)
+            slices.append(current_slice)
+        else:
+            current_slice.append(item)
+
+    return slices
+
+def read_file(file_path):
```

Comment:
Please add a comment what is the output of this function. Since read file can mean a lot of things, and here the output is the actual lines of the file (a list of strings if I'm not mistaking).